spec/systems/tasks_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'rails_helper'
+
+RSpec.describe 'tasks', type: :system do
+  let(:user) { create(:user) }
+  let(:task) { create(:task) }
+
+  describe 'ログイン前' do
+    describe 'タスクページ移管確認' do
```

Comment:
移管だと少し分かりづらい印象を受けたので、このくらいの記載で良いかもしれません。
```suggestion
    describe 'タスクページ確認' do
```

Reply:
9ee559b